src/app/services/dishes.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable, BehaviorSubject } from 'rxjs';
+import { map, tap } from 'rxjs/operators';
+import { environment } from '../environments/environment';
+import { FoodItemInterface } from '../shared/interfaces/food-item.interface';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class DishesService {
+  private baseUrl = environment.API_URL;
```

Comment:
not needed variable, instead create 
```suggestion
  private dishesUrl = `${environment.API_URL}/dishes`;
```